tests/test_fft.py

```diff
@@ -391,11 +385,17 @@ def test_hfft_1D(self, dtype, n, norm):
         assert_dtype_allclose(result, expected, check_only_type_kind=True)
 
     @pytest.mark.parametrize("dtype", get_complex_dtypes())
-    @pytest.mark.parametrize("n", [None, 5, 20])
+    @pytest.mark.parametrize("n", [None, 5, 18])
     @pytest.mark.parametrize("norm", ["forward", "backward", "ortho"])
     def test_hfft_1D_complex(self, dtype, n, norm):
         x = dpnp.linspace(-1, 1, 11)
         a = dpnp.sin(x) + 1j * dpnp.cos(x)
+        # input should be Hermitian
+        a[0].imag = 0
+        if n in [None, 18]:
+            f_ny = -1 if n is None else n // 2  # Nyquist mode
+            a[f_ny].imag = 0
+            a[f_ny:] = 0  # no data needed after Nyquist mode
```

Comment:
The lines look repeating by several tests. What do you think to decouple them to separate internal func?